src/nvim/eval/typval.h

```diff
@@ -426,6 +426,18 @@ static inline void tv_list_ref(list_T *const l)
   l->lv_refcount++;
 }
 
+/// Set a list as the return value
+///
+/// @param[in] tv   Object to receive the list
+/// @param[in] l    List to pass to the object
+static inline void tv_list_set_ret(typval_T *const tv, list_T *const l)
+  FUNC_ATTR_NONNULL_ARG(1)
```

Comment:
Check out other functions, this is how you mark *non-inline* ones. There is no declarations generation for headers (yet), so it is different.

Reply:
I don't know what you mean. Use `REAL_FATTR_`?